intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -18,15 +18,19 @@ \chapter{Running batch jobs}
 nodes} of the cluster. There you can prepare the work you want to get done on
 the cluster by, e.g., installing or compiling programs, setting up data sets,
 etc. The computations however, should not be performed on this login node. The
-actual work is done on the cluster's \strong{compute nodes}. These compute
-nodes are managed by the job scheduling software (Moab) and a Resource Manager
-(TORQUE), which decides when and on which compute nodes the jobs can run. It is
-usually not necessary 
+actual work is done on the cluster's \strong{compute nodes}. Each compute node
+contains a number of CPU \strong{cores}. The compute nodes are managed by the
+job scheduling software (Moab) and a Resource Manager (TORQUE), which decides
```

Comment:
Should be `Torque` and `MOAB`, I think...

Reply:
Yes, this was a pre-existing error.

Reply:
Hm, it seems it is correct like it is.

Reply:
Hmm, OK, nevermind me then. :)